proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -0,0 +1,103 @@
+# Project Name
+
+## Project Champion
+
+Who is the main point of contact during the application process?
+
+## CPC Champion (optional)
+
+If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
+
+## Description
+
+A rough description of the project
+
+## Statement of alignment with OpenJS Foundation charter and mission
+
+Please refer to [TBD][]
+
+## Current Code of Conduct
+
+Include a link
+
+## Current Project License
+
+Include a link
+
+## Current Source Control
+
+Link to Github or other source control
+
+## Issues Tracker
+
+Link to GitHub or other tracker
+
+## External Dependencies
+
+Full list of dependencies in your source tree including licenses.
+
+## Release Methodology and mechanics
+
+How often do you release and where are the releases made. Do you have an LTS program?
+
+## List of Committers
+
+Full list of individuals with a commit bit to your project and their corporate affiliation if applicable.
+
+## Governance Structure
+
+* Is there a leadership team?
+* Who are the members of the leadership team?
+* How are members of the leadership team nominated
+* How are commiters nominated?
+
+## Desired Initial Project Phase
+
+At Large / Growth / Impact
+
+## Official Communication Channels
+
+List current channels e.g.: Slack / IRC / Mailing lists
+
+## Project Website
+
+Include link
+
+## Social Media Accounts
+
+Links to social media accounts
+
+## Existing Financial Sponsorship
+
+Does your project currently receive funds? Who do they come from and what are the funds used for?
+
+## Infrastructure Needs or Requests
+
+What needs will your project have from the foundation?
```

Comment:
Could be helpful here to have a list of possible needs the Foundation can fulfill. Extra points if we know how asking for an item would affect the New Project Application's chances or pace of progression :)

Reply:
Comment resolved during #168